tests/integrations/subprocess/test_Subprocess__Popen.py

```diff
@@ -39,6 +41,52 @@ def test_call_with_path_arg(mock_sub, capsys, tmp_path):
     assert capsys.readouterr().out == ""
 
 
+@pytest.mark.parametrize("platform", ["linux", "darwin", "win32"])
+@pytest.mark.parametrize("start_new_session", [None, True, False])
+def test_call_with_start_new_session(mock_sub, capsys, platform, start_new_session):
+    "start_new_session is passed thru on Linux and macOS but converted for Windows"
+
+    mock_sub.command.sys.platform = platform
+    mock_sub.Popen(['hello', 'world'], start_new_session=start_new_session)
+
+    if platform == "win32":
+        mock_sub._subprocess.Popen.assert_called_with(
+            ['hello', 'world'],
+            text=True,
+            **({'creationflags': 0x8000200} if start_new_session is True else {}),
```

Comment:
It's preferable to avoid logic in tests as much as possible; this can be handled as an extra "popen_kwargs" parameter in `parametrize()` (with an empty dictionary for the non-win32 case)

Reply:
Incorporated in to `parametrize`.